src/core/ws.ts

```diff
@@ -13,6 +13,15 @@ import { WebSocketClientManager } from './ws/WebSocketClientManager'
 
 const wsBroadcastChannel = new BroadcastChannel('msw:ws-client-manager')
 
+// Allows the Node.js thread to exit if it is the only active handle in the event system.
+// https://nodejs.org/api/worker_threads.html#broadcastchannelunref
+if ('unref' in wsBroadcastChannel) {
+  const nodeBroadcastChannel = wsBroadcastChannel as BroadcastChannel & {
```

Comment:
You can achieve a better typing and overall usage experience by creating a custom predicate function for `unref`:

```ts
function hasUnrefMethod(channel: BroadcastChannel): channel is BroadcastChannel & { unref(): void } {
  return typeof Reflect.get(channel, 'unref') !== 'undefined'
}

const channel = new BroadcastChannel('')
if (hasUnrefMethod(channel)) {
    channel.unref()
}
```
> Double-check the actual call signature of `unref()` in Node.js to make sure we are annotating it correctly in the predicate function.

Reply:
Nice suggestion, I implemented it.